[PROF-13394] Improve setup source code integration documentation (#33671)
DataDog/documentation: content/en/profiler/enabling/dotnet.md

     <strong>Note:</strong> Use <code>stop</code> and <code>start</code> commands. A reset or restart does not always work.
   </div>

5. Optional: Set up [Source Code Integration][2] to connect your profiling data with your Git repositories.

> 6. A minute or two after starting your application, your profiles appear on the [Datadog APM > Profiler page][1].

Reviewer: ```suggestion
6. A couple of minutes after you start your application, your profiles appear on the [Datadog APM > Profiler page][1].
```
